modules/deployment/src/main/java/org/eclipse/xpanse/modules/deployment/deployers/terraform/TerraformDeployment.java

```diff
@@ -34,7 +35,13 @@ public class TerraformDeployment implements Deployment {
     public static final String VERSION_FILE_NAME = "version.tf";
     public static final String SCRIPT_FILE_NAME = "resources.tf";
 
-    private static final String WORKSPACE = "xpanse_deploy_ws";
+
+    private final String workspaceDirectory;
+
+    public TerraformDeployment(
+            @Value("${terraform.workspace.directory}") String workspaceDirectory) {
```

Comment:
We can use @Value("${terraform.workspace.directory:xpanse_deploy_ws}") set a default value

Reply:
@baixinsui  the default is already there in application.properties. 
My idea was to keep all default in application.properties and override it with any specific value via environment variables or via plugin specific property files. 
Do you see any issue with this approach?

Reply:
It’s OK. Set the default value just in case of losing the configuration item in application.properties.

Reply:
@baixinsui ok got it. done. Have added it now.